app/src/main/java/com/jerboa/ui/components/post/PostActivity.kt

```diff
@@ -305,7 +309,7 @@ fun PostActivity(
                     is ApiState.Failure -> ApiErrorText(postRes.msg)
                     is ApiState.Success -> {
                         val postView = postRes.data.post_view
-
+                        appState.addReturn(PostViewReturn.POST_VIEW, postView.copy(read = true))
```

Comment:
The anon check needs to be still there. Read feature is only available to real accounts